charms/kfp-ui/src/components/pebble_components.py

```diff
@@ -37,7 +37,8 @@ def get_layer(self) -> Layer:
                     self.service_name: {
                         "override": "replace",
                         "summary": "entry point for ml-pipeline-ui",
-                        "command": "node dist/server.js ../client/ 3000",  # Must be a string
+                        # command should be updated each time we switch from upstream to ROCK image
```

Comment:
Should we add the command from the rock and the command from upstream in the comment just for completeness?

Reply:
Good idea, added!